ui/src/table.tsx

```diff
@@ -294,7 +294,7 @@ const
   },
   ContextualMenu = ({ onFilterChange, col, listProps, selectedFilters, setFiltersInBulk }: ContextualMenuProps) => {
     const
-      isFilterChecked = (data: S, key: S) => !!selectedFilters && selectedFilters[data]?.includes(key),
+      isFilterChecked = (data: S, key: S) => !!selectedFilters && selectedFilters[data]?.includes(key) ? true : false,
```

Comment:
Small stylistic update. `!!` is a boolean conversion which turns `undefined` to `false` since it's a falsy value.

```suggestion
      isFilterChecked = (data: S, key: S) => !!selectedFilters && !!selectedFilters[data]?.includes(key),
```

Reply:
Done! I generally lean towards explicit return of true/false, but since the first part of this AND statement uses a boolean conversion then I can see that it makes sense to use it for both sides of the AND. Thanks for the suggestion!